Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -33,20 +35,66 @@ public GameObject ThumbRoot
             }
         }
 
-        [Range(0, 1)]
+        #region MyValue 
```

Comment:
For this region, is there something more descriptive than "MyValue" that could be used here? I see that this is covering the "MyValue" property (which, itself is also a little vague in terms of its naming and could probably have a more specific thing).

I'm still reading through this so just leaving this comment for now, but will hopefully come back with a suggestion.